raui-core/src/messenger.rs

```diff
@@ -73,6 +75,8 @@ impl<'a> Messenger<'a> {
     }
 }
 
+/// Macro for implementing [`MessageData`]. You may prefer to use the [derive
+/// macro][`macro@crate::MessageData`] instead.
```

Comment:
add: "but in case of auto-implementing MessageData for remote or std types, this might be the macro you find useful"